src/main/kotlin/calculator/Operator.kt

```diff
@@ -1,19 +1,26 @@
 package calculator
 
-interface Operator {
-    val value: String
+enum class Operator(val value: String) {
```

Comment:
코틀린은 함수를 일반 값처럼 다룰 수 있습니다. 이를 일급 시민(first-class citizen)인 함수라고 해요. 아래의 코드로 개선해 보면 어떨까요?

```suggestion
enum class Operator(val value: String, val operate: (Int, Int) -> Int) {
```